models/token.js

```diff
@@ -0,0 +1,13 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../config/config");
+const user = require("./user");
+const token = sequelize.define("token", {
+  token: {
+    type: DataTypes.STRING,
+  },
+});
+
+user.hasOne(token, {foreignKey: "userId", as: "token" });
```

Comment:
manage this association accordingly because you are creating token on every login